.github/workflows/cicd.yml

```diff
@@ -18,20 +18,19 @@ jobs:
       - id: set-packages
         run: echo "::set-output name=packages::$(find ./packages -maxdepth 2 -type f -name "pubspec.yaml" | jq -R -s -c 'split("\n")[:-1]')"
 
-  dartdoc:
-    runs-on: ubuntu-latest
-    needs: [get-packages-paths]
-    strategy:
-      matrix:
-        package: ${{fromJson(needs.get-packages-paths.outputs.packages)}}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: subosito/flutter-action@v1
-        with:
-          channel: 'stable'
-      - uses: flame-engine/flame-dartdoc-action@v2
-        with:
-          package-path: ${{ matrix.package }}
+  #dartdoc:
```

Comment:
:scream:

Reply:
Just trying to get stuff to work, it's not going to be commented out later :)